doc/docs/master/concepts/pipeline-concepts/datum/index.md

```diff
@@ -5,63 +5,64 @@
     all data at once, each directory independently, individual
     files one by one, or combined data from multiple inputs together.
 
-A datum is the smallest indivisible unit of computation within a
-job.
+## Definition
+A datum is **the smallest indivisible unit of computation within a
+job**.
 A job can have one, many or no datums. Each datum is processed
-independently with a single execution of the user code and
-then the results of all the datums are merged together to
+independently with a single execution of the user code on one of
+the pipeline worker nodes.
+The results of all the datums are then combined together to
 create the final output commit.
 
-TODO: Inputs are based on branches, not repositories. Although that may
-be a common way to think about it since people will typically just use the master branch as input,
-it is an important distinction that should be clear in the docs.  
+## Glob Pattern and Data distribution
+Think of datums as a way to **divide your input data** 
+and **distribute processing workloads**.
+They are instrumental in optimizing your pipeline performance.
 
-A datum defines the input data. An input can take one or multiple
-repositories. Pachyderm has the following types of inputs that
-combine multiple repositories:
+You define how your data is spread among workers by
+**specifying a [glob pattern](glob-pattern.md) for each input** of your pipeline. 
+The data in the `input` of your pipeline is "glob-ed" or "broken down" in datums,
```

Comment:
I think it would be better if we had one section for PFS inputs that included all of the glob information / examples. Glob patterns only apply to PFS inputs, and should only really be discussed in that context imo.